hooks/tests/hook_test_green.py

```
@@ -0,0 +1,33 @@
+        self.connection = None
+
+    def connect(self):
+        try:
+            import psycopg2
+
+            self.connection = psycopg2.connect(
+                host=self.host, port=self.port, database=self.database
+            )
+            return self.connection
+        except Exception as e:
+            raise ConnectionError(f"Failed to connect: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)